syntax/grammar.mjs

```diff
@@ -111,19 +111,33 @@ let Attribute = defer(() =>
         maybe(inline_space),
         char("="),
         maybe(inline_space),
-        Pattern.abstract)
+        Value.abstract)
```

Comment:
I think we should undo this part, attributes in general don't have an `XAttributeVariantExpression`, independent of Message or Term.

Reply:
Good point, thanks. I think #128 will make this easier to add in the future, because the grammar won't need to enumerate each of the possible `X` in `X{Attribute,Variant}Expression`.

Reply:
In fact, I could also keep the `Pattern` in `Message`, too. Do you have a preference wrt. checking the values of Messages and Attributes in `grammar` vs. in `abstract`?

Reply:
I agree. I think `grammar` would be a good place. That makes the variants for terms stick out more.